### PR TITLE
Added a 4 Note Auto and Automated some processes

### PR DESCRIPTION
**Pull request by isaiah-g04**
I drew and fine tuned a 4 note auto using the inside notes, also fixed some of the automating processes

### CODE REVIEW

File: src/main/java/frc/robot/Constants.java
Comment (Nixis198) [inline src/main/java/frc/robot/Constants.java:1]: Any reason why you updated this? Pathplanner doesn't use theses.

File: src/main/java/frc/robot/RobotContainer.java [hunk 1]
```diff
@@ -1,5 +1,44 @@
 package frc.robot;
 
+import com.pathplanner.lib.auto.AutoBuilder;
+import com.pathplanner.lib.auto.NamedCommands;
+
```
Comment (Nixis198) [inline src/main/java/frc/robot/RobotContainer.java:5]: Did VSCode add all of theses imports. Right below them are the same imports.

**Review by Nixis198 — CHANGES_REQUESTED** [review]
If you can, reduce the imports in RobotContainer.java. If not, just submit again.

File: src/main/java/frc/robot/RobotContainer.java (hunk 1 above)
Comment (isaiah-g04) [reply src/main/java/frc/robot/RobotContainer.java:5]: Yes VSCode did, I didn't

Comment (Nixis198) [reply src/main/java/frc/robot/RobotContainer.java:5]: Lmao

File: src/main/java/frc/robot/Constants.java
Comment (isaiah-g04) [reply src/main/java/frc/robot/Constants.java:1]: I updated these values in the pathplanner app, but it didn't seem to change the speed. I updated these ones and it did change speed, it does use these, look under the autobuilder PID values

Comment (Nixis198) [reply src/main/java/frc/robot/Constants.java:1]: The only place I am seeing these used are in [exampleAuto.java](https://github.com/HoltRobotics/2024RobotV2/blob/main/src/main/java/frc/robot/autos/exampleAuto.java#L24).
We can take a look at it tomorrow.

Comment (isaiah-g04) [reply src/main/java/frc/robot/Constants.java:1]: image

**Pull request merged**